test/perf/tests/naked-styled-components/index.js

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { createMuiTheme } from '@material-ui/core/styles';
+import { spacing } from '@material-ui/system';
+import styledComponents, {
+  ThemeProvider as StyledComponentsThemeProvider,
+} from 'styled-components';
+import { logReactMetrics } from '../utils';
+
+const materialSystemTheme = createMuiTheme();
+const NakedStyleComponents = styledComponents('div')(spacing);
```

Comment:
Can we stop calling these "naked"? Seems like it isn't even unstyled.

Reply:
I am just copying the examples from `material-ui-benchmarks`, will reiterate on all examples once I start working on the system updates. Would like to keep these changes minimal.

Reply:
That works too.